### PR TITLE
Ring UI: No more useless `WithClassName`

### PR DESCRIPTION
**Pull request by aerialist7**
cc @turansky @sgrishchenko

### CODE REVIEW

File: kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt [hunk 1]
```diff
@@ -5,13 +5,13 @@ package ringui
...
-import react.dom.WithClassName
+import react.PropsWithClassName
 
-external interface GridProps : PropsWithChildren
+external interface GridProps : PropsWithChildren, PropsWithClassName
 
 external val Grid: ComponentClass<GridProps>
 
-external interface RowProps : GridProps, WithClassName {
+external interface RowProps : GridProps, PropsWithClassName, PropsWithChildren {
```
Comment (turansky) [inline kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt:14]: Duplication

File: kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt [hunk 2]
```diff
@@ -28,7 +28,7 @@ external interface RowProps : GridProps, WithClassName {
 
 external val Row: ComponentClass<RowProps>
 
-external interface ColProps : GridProps, WithClassName {
+external interface ColProps : GridProps, PropsWithClassName, PropsWithChildren {
```
Comment (turansky) [inline kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt:31]: Duplication

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt (hunk 1 above)
Comment (aerialist7) [reply kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt]: Fixed

File: kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt (hunk 2 above)
Comment (aerialist7) [reply kotlin-ring-ui/src/main/kotlin/ringui/Grid.kt]: Fixed

**Review by turansky — APPROVED** [review]

**Comment by Leonya** [comment]
Released in pre.239

**Pull request merged**